hooks/webhook.go

```diff
@@ -101,3 +99,19 @@ func webhookURL(webhook *v1alpha1.Webhook) (string, error) {
 	}
 	return fmt.Sprintf("%s://%s.%s:%v%s", protocol, webhook.Service.Name, webhook.Service.Namespace, port, *webhook.Path), nil
 }
+
+func webhookTimeout(webhook *v1alpha1.Webhook) (time.Duration, error) {
+	zero := time.Duration(0)
+
+	if webhook.Timeout == nil {
+		// Defaults to 10 Seconds to preserve current behavior.
+		return 10 * time.Second, nil
+	}
+
+	if (*webhook.Timeout).Duration <= zero {
```

Comment:
I think it should work as just `webhook.Timeout.Duration` since `.` dereferences as needed. Same below.

Reply:
Changed